tests/test_1399-from-jax.py

```diff
@@ -6,9 +6,10 @@
 import awkward as ak  # noqa: F401
 
 jax = pytest.importorskip("jax")
-
 jax.config.update("jax_enable_x64", True)
 
+ak.jax.register_and_check()
```

Comment:
Just last week, you put in some effort to ensure that the tests are all thread-safe. If two of these `ak.jax.register_and_check` functions start at the same time, would they interfere with each other? I'll go check.